docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -276,6 +276,8 @@ These parameters define the permissions required in order to access the associat
 
 [discrete]
 === Network
+It is not possible to define directly the ID of the {aws} VPC the Elastic Serverless Forwarder lambda belongs to.
+The {aws} VPC is defined instead by its security group IDs and subnet IDs. 
```

Comment:
It might be worth explaining why it is not possible and add a link to the AWS docs.

As additional note, I do not think this information is enough to close out https://github.com/elastic/elastic-serverless-forwarder/issues/488 as there are more pieces involved in making ESF work with PrivateLink, apart from the VPC settings (e.g VPC endpoints)

Reply:
VPC endpoints are documented [here](https://github.com/elastic/elastic-serverless-forwarder/blob/3fd63d98e41143f6bc34d0489607353f3c5a2b30/docs/en/aws-serverless-troubleshooting.asciidoc#prerequisites-when-attached-to-a-vpc)

Let's move the content to this section! (and add the requirement for an EC2 VPC endpoint if cloudwatch logs are used as input)

I will add a link to https://www.elastic.co/guide/en/cloud/current/ec-traffic-filtering-vpc.html as well

do you have any hint about what @Udayel mentioned in #488?
> document few important parameters like `BOTO_DISABLE_COMMONNAME=False`

I remember it was something related to a version bump of the ESF boto dependency, but I never hit the issue and I'm not sure if it still applies